### PR TITLE
Refactor FXIOS-8313 [Multi-window] SwiftUI view themeing

### PR DESCRIPTION
**Pull request by mattreaganmozilla**
## :scroll: Tickets
[Jira ticket](https://mozilla-hub.atlassian.net/browse/FXIOS-8313)
[Github issue](https://github.com/mozilla-mobile/firefox-ios/issues/18426)

## :bulb: Description

Updates SwiftUI views so that themeing works with multi-window. _(Note: Draft only updates one file, but the others will follow the same basic pattern)_

## :pencil: Checklist
You have to check all boxes before merging
- [ ] Filled in the above information (tickets numbers and description of your work)
- [ ] Updated the PR name to follow our [PR naming guidelines](https://github.com/mozilla-mobile/firefox-ios/wiki/Pull-Request-Naming-Guide)
- [ ] Wrote unit tests and/or ensured the tests suite is passing
- [ ] When working on UI, I checked and implemented accessibility (minimum Dynamic Text and VoiceOver)
- [ ] If needed, I updated documentation / comments for complex code and public methods
- [ ] If needed, added a backport comment (example `@Mergifyio backport release/v120`)

### CODE REVIEW

**Review by adudenamedruby — APPROVED** [review]
LGTM

**Review by cyndichin — APPROVED** [review]
looks good to me! thanks for breaking up the changes :)

**Comment by mattreaganmozilla** [comment]
Remaining SwiftUI views have been updated (cc @thatswinnie, several of the Fakespot files were updated. LMK if any concerns, and any 👀 you can lend to the review are appreciated. Thank you)

File: firefox-ios/Client/Frontend/Fakespot/Views/CircularProgressView.swift [hunk 1]
```diff
@@ -6,8 +6,9 @@ import SwiftUI
 import Common
 
 struct CircularProgressView: View, ThemeApplicable {
-    @Environment(\.themeType)
-    var themeVal
+    let windowUUID: WindowUUID
```
Comment (dataports) [inline firefox-ios/Client/Frontend/Fakespot/Views/CircularProgressView.swift:9]: I think this is already defined in `FakespotMessageCardViewModel`

**Review by dataports — APPROVED** [review]
Left one small comment, otherwise looks great 👍

File: firefox-ios/Client/Frontend/Fakespot/Views/CircularProgressView.swift (hunk 1 above)
Comment (mattreaganmozilla) [reply firefox-ios/Client/Frontend/Fakespot/Views/CircularProgressView.swift:9]: Thanks for catching 👍, I've added a ToDo, will clean this up in the next follow-up PR for 8313

**Pull request merged**